Messages/Messages/LoginViewController.swift

```diff
@@ -45,10 +45,12 @@ class LoginViewController: UIViewController, UITextFieldDelegate {
                 return
             }
             
-            let storyBoard: UIStoryboard = UIStoryboard(name: "Main", bundle: nil)
-            let newViewController = storyBoard.instantiateViewController(withIdentifier: "ChannelViewController") as! ChannelViewController
+            SessionCache.sharedInstance.user = loggedUser
+            UserListFacade.createUser(user: loggedUser)
```

Comment:
you should have a completion block to know the user was created before present the tab view controller